docs/source/user_guide/model_understanding.ipynb

```diff
@@ -425,7 +425,7 @@
    "source": [
     "The interpretation of the table is the same for regression problems - but the SHAP value now corresponds to the change in the estimated value of the dependent variable rather than a change in probability. For multiclass classification problems, a table will be output for each possible class.\n",
     "\n",
-    "This functionality is currently **not supported** for **XGBoost** models or **CatBoost multiclass** classifiers.\n",
+    "This functionality is currently **not supported** for **CatBoost multiclass** classifiers.\n",
```

Comment:
To confirm, this works for XGBoost multiclass too then? :D

Reply:
Yes! 🎉